contracts/contract_transactions.md

```diff
@@ -0,0 +1,68 @@
+[back](./contracts.md)
+## Contract Transactions
+
+Contract transactions are of three types:
+- Create
+- Call
+- Disable
+
+A contract is also a normal account with a balance.
+
+
+### Create Contract Transaction
+
+Anyone can create a new contract by submitting a create contract transaction.
+
+The transaction contains:
+- The address of the contract owner (the one signing and paying for the transaction) 
+- The byte code of the contract
+- The VM to use
+- A transaction fee
+- Optionally: A initial call to the contract including name, args and gas. (Can be used to emulate init function of Ethereum/Solidity contracts)
```

Comment:
If the init fails, is the contract created?

Reply:
If it runs out of gas or similar I guess the contract creation should be considered to fail.